src/components/e-label.vue

```diff
@@ -116,11 +116,7 @@
     }
 
     &__name--invisible {
-      position: absolute;
-      overflow: hidden;
-      height: 0;
-      width: 0;
-      top: 0;
+      @extend %invisible;
```

Comment:
please change to be a mixin, as we are getting rid of any extends in the PR #182 because extends are not supported with @use and @forward